[2021-01-15 16:46:44,321] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: noti_test.op3 2021-01-15T07:43:50.673128+00:00 [queued]>
[2021-01-15 16:46:44,328] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: noti_test.op3 2021-01-15T07:43:50.673128+00:00 [queued]>
[2021-01-15 16:46:44,329] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-01-15 16:46:44,329] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2021-01-15 16:46:44,329] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-01-15 16:46:44,334] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): op3> on 2021-01-15T07:43:50.673128+00:00
[2021-01-15 16:46:44,336] {standard_task_runner.py:54} INFO - Started process 9561 to run task
[2021-01-15 16:46:44,387] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'noti_test', 'op3', '2021-01-15T07:43:50.673128+00:00', '--job_id', '60', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/noti_test.py', '--cfg_path', '/var/folders/4f/xnn880bn51d9945mdydbgwdw0000gn/T/tmpjn04sirc']
[2021-01-15 16:46:44,388] {standard_task_runner.py:78} INFO - Job 60: Subtask op3
[2021-01-15 16:46:44,409] {logging_mixin.py:112} INFO - Running <TaskInstance: noti_test.op3 2021-01-15T07:43:50.673128+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-01-15 16:46:44,422] {logging_mixin.py:112} INFO - function3
[2021-01-15 16:46:44,422] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-01-15 16:46:49,324] {local_task_job.py:102} INFO - Task exited with return code Negsignal.SIGABRT
